### DISCUSSION TITLE
[Suggested feature] Specify a fixed seed when obfuscating

### DISCUSSION THREAD

**Discussion by DeboBurro** [dashingsoft/pyarmor#1631] [Ideas]
## Motivation
The obfuscation part has a random seed that makes our build artifacts different from each other even if they were built from the same commit. It makes us hard to identify if it's build process or our software logic problem when a release has issues.

## Proposed idea
Add an argument that we can pass in our own fixed seed. Ex: `pyarmor gen --seed 123 --output /tmp/dist aa.py`

**Comment by jondy** [comment]
I don't know what do you mean "our build artifacts different from each other". Could you explain it with an simple example and shell scripts to make it clear?

**Comment by jondy** [comment]
Maybe use shared runtime package could get your demands, refer to command `pyarmor gen runtime` in the man page
